docs/best-practices/systables.txt

```diff
@@ -225,7 +225,45 @@ list the hosts and tables with the highest number of rows inside a single
 shard. There's a lot of additional information available for shards. To find
 out more about that, see our :ref:`documentation <import_shards_replicas>`.
 
-Step 5: Managing Snapshots
+
+Step 5: Analyze Allocation Problems
+===================================
+
+Related to the previous step about gaining insights about shards and
+replication is the step about cluster wide shard allocations.
+
+In some circumstances shards allocations behave different than you might expect
+to behave. A typical example is that a table remains underreplicated for some
+not obvious reason. Then you want to know the decisions of the cluster not to
```

Comment:
"Then you want to know the decisions of the cluster not to allocate the shards"

this still doesn't parse well for me. can you rephrase? do you mean "you may want to know why the cluster has not allocated the shards" or "you may want to find out what is causing the cluster not to.."?

Reply:
you want to know the decisions the cluster made. isn't that the same as "why the cluster did not allocate shards" ?
do you have a suggestion for the wording?